### ISSUE TITLE
Progress does not work on android

### ISSUE THREAD

**Issue by mannol** [oblador/react-native-image-progress#48]
Progress does not work on android because `onProgress` Image callback is iOS **only**.

**Comment by oblador** [comment]
Yep, we just have to wait for `react-native` core to support it 😄

**Comment by mannol** [comment]
Yeah, it's a shame it doesn't still...

**Comment by oblador** [comment]
Hey, latest version of `react-native-fast-image` should support this. Check the readme of this repo for usage 👍 Let me know if it works for you!

**Comment by mannol** [comment]
Oh my! That would be awesome
